pkg/apis/core/validation/validation_test.go

```diff
@@ -1420,7 +1420,7 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 		toggleBlockVolumeFeature(scenario.enableBlock, t)
 		scenario.oldClaim.ResourceVersion = "1"
 		scenario.newClaim.ResourceVersion = "1"
-		errs := ValidatePersistentVolumeClaimUpdate(scenario.newClaim, scenario.oldClaim)
```

Comment:
Validation should not be mutating an object.

Reply:
ValidateObjectMetaUpdate, I think

Reply:
The status copy over to new pvc has been there forever. I dunno if we can safely remove it. cc @saad-ali

Reply:
> ValidateObjectMetaUpdate, I think
 
@liggitt I am not sure I understand.